tsl/org.testeditor.tsl.dsl.tests/src/org/testeditor/tsl/dsl/parser/DescriptionParserTest.xtend

```diff
@@ -0,0 +1,51 @@
+package org.testeditor.tsl.dsl.parser
+
+import org.junit.Test
+
+class DescriptionParserTest extends AbstractParserTest {
+
+	@Test
+	def void parseSimpleDescription() {
+		// given
+		val description = 'simple description'
+
+		// expect
+		description.parseDescription.assertEquals(description)
+	}
+
+	@Test
+	def void parseDescription() {
+		// given
+		val description = '''
+			Feature: Sample feature
+				As a developer I want the sample feature in order
```

Comment:
is '*' allowed within the text? if so it could be useful to test it.
if '*' is not allowed a test showing that would help too, right?